cmdstanpy/utils.py

```diff
@@ -933,6 +935,20 @@ def __init__(self, *objs: Union[str, dict], logger: logging.Logger = None):
                 if not os.path.exists(obj):
                     raise ValueError("File doesn't exist {}".format(obj))
                 self._paths[i] = obj
+            elif isinstance(obj, list):
+                if not all(isinstance(obj_item, str) for obj_item in obj):
+                    raise ValueError("Not all list items are str")
```

Comment:
Maybe a bit better error msg?

Reply:
yes.  error messages should correspond to how user views these objects.  how about:

`'List element {} must be a filename string, found {}'.format(i, obj_item)`